torch/csrc/jit/passes/tensorexpr_fuser.cpp

```diff
@@ -1159,9 +1163,33 @@ class TensorExprFuser {
     }
   }
 
+  // This function parses the option provided by the environment variable
+  // "PYTORCH_TENSOREXPR_DONT_FUSE".
+  // This variable allows users to disable fusion on a list of specified
+  // operators that are separated by ':'. e.g.,
+  // 'PYTORCH_TENSOREXPR_DONT_FUSE="clamp:mul:add"' disables fusion on
+  // aten::clamp, aten::mul and aten::add.
+  std::set<NodeKind> parseTENotFuseOption() {
```

Comment:
Since this is initializing a member, I'd probably make it `void` and not return anything.  I guess alternatively you could have it construct a set as a local variable, return that, and put it in the initialization list of the constructor.  I'm just wary of a function returning a value that is probably thrown away :-)